pyproject.toml

```
@@ -1,10 +1,66 @@
 [build-system]
 requires = [
     "pkgconfig",
-    "setuptools >= 46.4.0",
+    # setuptools >= 67.0 required for Python 3.12+ support
+    # Next step should be >= 77.0 for PEP 639 support
+    # Don't bump to early to give distributors time to update
+    # their setuptools version.
+    "setuptools >= 67.0",
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+name        = "aiohttp"
+license     = {text = "Apache-2.0 AND MIT"}
+description = "Async http client/server framework (asyncio)"
+readme      = "README.rst"
+maintainers = [{name = "aiohttp team", email = "team@aiohttp.org"}]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Framework :: AsyncIO",
+  "Intended Audience :: Developers",
+  "Operating System :: POSIX",
+  "Operating System :: MacOS :: MacOS X",
+  "Operating System :: Microsoft :: Windows",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+  "Topic :: Internet :: WWW/HTTP",
+]
+requires-python = ">= 3.9"
+dynamic = ["version", "dependencies", "optional-dependencies"]
```

Let's have this multi-lined and ordered:
```suggestion
dynamic = [
  "dependencies",
  "optional-dependencies",
  "version",
]
```